src/lib/network_pool/transaction_pool.ml

```diff
@@ -1572,6 +1572,32 @@ let%test_module _ =
         global_slot_since_genesis = Mina_numbers.Global_slot_since_genesis.zero
       }
 
+    module Test_Account = struct
+      type t =
+        { name : string
+        ; balance : string
+        ; timing : Mina_base.Account_timing.t
+        ; permissions : Permissions.t
+        ; zkappify : bool
+        }
+
+      let default =
+        { name = ""
+        ; balance = "900000000.0"
+        ; timing = Mina_base.Account_timing.Untimed
+        ; permissions = Permissions.user_default
+        ; zkappify = false
+        }
+    end
+
+    let test_accounts =
+      Array.mapi test_keys ~f:(fun i _ ->
+          if i mod 2 = 0 then Test_Account.default
+          else { Test_Account.default with zkappify = true } )
```

Comment:
Wouldn't it be better to have a Quickcheck generator for these? I understand that generating lots of fresh keypairs in tests might be needlessly intense computationally, but generating other data seems okay, while providing better versatility and coverage for the tests.

Reply:
You are right. I didn't want to make too many revolutions in the code. This is good point for improvements for this tests